.husky/pre-commit

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-yarn lint-staged
+npx lint-staged
```

Comment:
```suggestion
npm run lint-staged
```

Reply:
no it didn't recognize lint-staged as a script. Should I add it to the `scripts` in `package.json`?
```
"scripts": {
    ...
    "lint-staged": "npx lint-staged",
    ...
```

Reply:
Btw the error when trying your suggestion was:
```
0 verbose cli [
0 verbose cli   '/Users/michaelcohen/.nvm/versions/node/v16.11.0/bin/node',
0 verbose cli   '/Users/michaelcohen/.nvm/versions/node/v16.11.0/bin/npm',
0 verbose cli   'run',
0 verbose cli   'lint-staged'
0 verbose cli ]
1 info using npm@8.0.0
2 info using node@v16.11.0
3 timing npm:load:whichnode Completed in 0ms
4 timing config:load:defaults Completed in 4ms
5 timing config:load:file:/Users/michaelcohen/.nvm/versions/node/v16.11.0/lib/node_modules/npm/npmrc Completed in 0ms
6 timing config:load:builtin Completed in 0ms
7 timing config:load:cli Completed in 2ms
8 timing config:load:env Completed in 0ms
9 timing config:load:file:/Users/michaelcohen/Projects/stream-js/.npmrc Completed in 1ms
10 timing config:load:project Completed in 3ms
11 timing config:load:file:/Users/michaelcohen/.npmrc Completed in 2ms
12 timing config:load:user Completed in 2ms
13 timing config:load:file:/Users/michaelcohen/.nvm/versions/node/v16.11.0/etc/npmrc Completed in 1ms
14 timing config:load:global Completed in 1ms
15 timing config:load:validate Completed in 1ms
16 timing config:load:credentials Completed in 1ms
17 timing config:load:setEnvs Completed in 1ms
18 timing config:load Completed in 16ms
19 timing npm:load:configload Completed in 16ms
20 timing npm:load:setTitle Completed in 6ms
21 timing npm:load:setupLog Completed in 1ms
22 timing config:load:flatten Completed in 5ms
23 timing npm:load:cleanupLog Completed in 2ms
24 timing npm:load:configScope Completed in 0ms
25 timing npm:load:projectScope Completed in 0ms
26 timing npm:load Completed in 32ms
27 timing command:run-script Completed in 5ms
28 verbose stack Error: Missing script: "lint-staged"
28 verbose stack
28 verbose stack To see a list of scripts, run:
28 verbose stack   npm run
28 verbose stack     at RunScript.run (/Users/michaelcohen/.nvm/versions/node/v16.11.0/lib/node_modules/npm/lib/run-script.js:104:13)
29 verbose cwd /Users/michaelcohen/Projects/stream-js
30 verbose Darwin 21.4.0
31 verbose argv "/Users/michaelcohen/.nvm/versions/node/v16.11.0/bin/node" "/Users/michaelcohen/.nvm/versions/node/v16.11.0/bin/npm" "run" "lint-staged"
32 verbose node v16.11.0
33 verbose npm  v8.0.0
34 error Missing script: "lint-staged"
34 error
34 error To see a list of scripts, run:
34 error   npm run
35 verbose exit 1
```

Reply:
Kk, lets keep as is for now.